src/services/ShopwareDataHelpers.ts

```diff
@@ -105,27 +105,6 @@ export const getPaymentMethodId = async (adminApiContext: AdminApiContext, handl
     return result.data[0].id;
 };
 
-/**
- * Gives the default shipping method back called Standard
- * @param adminApiContext - An AdminApiContext entity
- */
-export const getDefaultShippingMethodId = async (adminApiContext: AdminApiContext): Promise<string> => {
```

Comment:
We should deprecate this method (otherwise it would be a break) and create the more general `getShippingMethodId`, that takes the name as the parameter.